dataplane/cmd/redbeams.go

```diff
@@ -15,32 +15,92 @@ import (
 func init() {
 	DataPlaneCommands = append(DataPlaneCommands, cli.Command{
 		Name:  "redbeams",
-		Usage: "manage RBDMS instances and databases",
+		Usage: "manage relational databases and database servers",
 		Subcommands: []cli.Command{
 			{
 				Name:  "dbserver",
-				Usage: "manage redbeams instances",
+				Usage: "manage redbeams database servers",
 				Subcommands: []cli.Command{
 					{
 						Name:   "list",
-						Usage:  "list all RBDMS instances",
+						Usage:  "list all database servers",
 						Before: cf.CheckConfigAndCommandFlagsWithoutWorkspace,
-						Flags:  fl.NewFlagBuilder().AddFlags(fl.FlEnvironmentCrn, fl.FlWaitOptional).AddAuthenticationFlags().Build(),
-						Action: redbeams.ListRBDMSInstances,
+						Flags:  fl.NewFlagBuilder().AddFlags(fl.FlEnvironmentCrn).AddAuthenticationFlags().AddOutputFlag().Build(),
+						Action: redbeams.ListDatabaseServers,
 						BashComplete: func(c *cli.Context) {
-							for _, f := range fl.NewFlagBuilder().AddFlags(fl.FlEnvironmentCrn, fl.FlWaitOptional).AddAuthenticationFlags().Build() {
+							for _, f := range fl.NewFlagBuilder().AddFlags(fl.FlEnvironmentCrn).AddAuthenticationFlags().AddOutputFlag().Build() {
+								fl.PrintFlagCompletion(f)
+							}
+						},
+					},
+					{
+						Name:   "get",
+						Usage:  "get a database server by CRN",
+						Before: cf.CheckConfigAndCommandFlagsWithoutWorkspace,
+						Flags:  fl.NewFlagBuilder().AddFlags(fl.FlCrn).AddAuthenticationFlags().AddOutputFlag().Build(),
+						Action: redbeams.GetDatabaseServer,
+						BashComplete: func(c *cli.Context) {
+							for _, f := range fl.NewFlagBuilder().AddResourceDefaultFlags().AddFlags(fl.FlCrn).AddAuthenticationFlags().AddOutputFlag().Build() {
+								fl.PrintFlagCompletion(f)
+							}
+						},
+					},
+					{
+						Name:   "getbyname",
+						Usage:  "get a database server by name",
+						Before: cf.CheckConfigAndCommandFlagsWithoutWorkspace,
+						Flags:  fl.NewFlagBuilder().AddFlags(fl.FlEnvironmentCrn, fl.FlName).AddAuthenticationFlags().AddOutputFlag().Build(),
+						Action: redbeams.GetDatabaseServerByName,
+						BashComplete: func(c *cli.Context) {
+							for _, f := range fl.NewFlagBuilder().AddResourceDefaultFlags().AddFlags(fl.FlEnvironmentCrn, fl.FlName).AddAuthenticationFlags().AddOutputFlag().Build() {
+								fl.PrintFlagCompletion(f)
+							}
+						},
+					},
+					{
```

Comment:
Since the status is present in the resource for GET, I am not sure this is needed. This is user facing, so waiting a bit more and getting a bit more chatty response in case of the GET is ok I think.

Reply:
We discussed this this morning. The status in the get (changing to describe) results is only present after a database server has been created, and this getstatus call is for during database server creation. This is kind of confusing. We're going to rework redbeams so that only a single get/describe endpoint is necessary. Until that happens, I'll leave the operations in place so that callers can get at the status information whenever they need it.